### ISSUE TITLE
create a video through the api with a download_url property

### ISSUE THREAD

**Issue by jvanaert** [jwplayer/jwplatform-node#6]
I'm trying to use this library to publish some video's through a script. Basicly my code is as follows:
```
const jwApi = new JWPlatformAPI({ apiKey: JWPLAYER_API_KEY, apiSecret: JWPLAYER_SECRET });

const video = "http://commondatastorage.googleapis.com/gtv-videos-bucket/sample/BigBuckBunny.mp4"

const jwResponse = await jwApi.videos.create({ title: 'my video', download_url: video });
```

As a response i get
```
{ jwResponse:
   { status: 'ok',
     media: { type: 'video', key: 'GTlEutz5' },
     link:
      { path: '/v1/videos/upload',
        query: [Object],
        protocol: 'http',
        address: 'upload.jwplatform.com' },
     rate_limit: { reset: 1573036920, limit: 60, remaining: 59 } } }
```

where I did not expect to receive an upload url, I did expect that the supplied video url was used to create a jw-hosted video (cfr. https://developer.jwplayer.com/jw-platform/docs/developer-guide/management-api/uploading-videos/)

Am I using this library in a wrong way to accomplish this ?

**Comment by boyntoni** [comment]
Hi @jvanaert, thanks for reporting this, and appreciate the feedback. You are not using the library in an incorrect fashion; your use case is the intended workflow.

We will take a look at this as soon as possible and follow up shortly with a solution.

**Comment by boyntoni** [comment]
@jvanaert Thanks again for filing the issue, and sorry for the issue. We have made the fix and deployed a new version (0.0.4) to NPM.

Please let us know if you have any other feedback.

**Comment by jvanaert** [comment]
works fine now. Thanks for the quick help !
